docs/appendices/release-notes/5.10.1.rst

```diff
@@ -44,6 +44,13 @@ See the :ref:`version_5.10.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed a recovery issue that led to records of a table with an ``OBJECT``
+  sub-column being part of a standalone ``FULLTEXT`` index to be never visible
+  for ``MATCH`` queries.
+  To mitigate the issue:
+  1. For non-composite indices: declare index next to a sub-column definition.
+  2. For composite indices: reindex the table.
```

Comment:
I will remove part for non-composite. It's a way to mitigate **before** indexing on old versions that don't have the fix